data/changelog/ocaml/2023-08-??-ocaml-5.1.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Release of OCaml 5.1.0
+description: Release of OCaml 5.1.0
+date: "2023-08-??"
+tags: [ocaml, release]
+changelog: |
+---
+
+We have the pleasure of celebrating the birthday of ??? by announcing the release of OCaml version 5.1.0.
+
+Some of the highlights in OCaml 5.1.0 are:
+
+   - Many runtime performance regression fixes (dynlinking, weak array, weak hash sets, GC with idle domains, GC prefetching).
+   - Restored support for RISC-V and s390x architectures.
+   - Restored Cygwin port.
+   - Reduced installation size (50% reduction)
+   - Compressed compilation artefacts (.cmi, .cmt, .cmti, .cmo, .cma files)
+   - More than 10 standard library functions made tail-recursive with Tail-Recursion-Modulo-Cons (TRMC)
+   - More examples in the standard library documentation
+   - More than 20 improved error messages
+
+OCaml 5.1.0 is still a relatively experimental release compared to the OCaml
+4.14 branch. In particular:
+
+   - The POWER port is being tested in the dev version of the compiler.
+   - The Windows MSVC port is still unavailable.
+   - Ephemeron performances need to be investigated.
+   - GC compaction is a work in progress.
+   - statmemprof is a work in progress.
+   - There are a number of known runtime concurrency bugs (that triggers under
+     rare circumstances) 
+
+We are planning to address those regressions, hopefully in the next release OCaml 5.2.0. Meanwhile, the OCaml 4.14 branch will be maintained and the next release on the OCaml 4.14 branch, OCaml 4.14.2, should follow this release in the upcoming months.
```

Comment:
I would prudently say that we are planning to address *some* of those regressions.

Reply:
`We are planning to address some of those regressions` can be read as `We already  know that some of those regression will never be fixed`. If the issue is the implicit 5.2 timeframe, I would propose:

> We are planning to address those regressions, hopefully in time for the next 5.2.0 release for some of them. 

?

Reply:
Have we reached consensus on all of these changes?

Reply:
Which part do you find non-consensual?